### PR TITLE
Feature/iot 1320 fix sig fox connection

### PR DESCRIPTION
**Pull request by GufCab**

### CODE REVIEW

**Review by AramAlsabti — CHANGES_REQUESTED** [review]
Might be a separate issue. After adding a SigFox connection and editing it (/edit-group), it throws this error irregardless of whatever's in the input fields. It seems to me that the organization id is missing in the request

billede

File: src/controllers/admin-controller/sigfox/sigfox-group.controller.ts [hunk 1]
```diff
@@ -81,7 +81,7 @@ export class SigfoxGroupController {
         try {
             group = await this.service.findOne(id);
         } catch (err) {
-            throw new NotFoundException();
+			return null;
```
[the remark this answers is not in the document]
Comment (AramAlsabti) [reply src/controllers/admin-controller/sigfox/sigfox-group.controller.ts:84]: And wouldn't it be more correct to still throw the error and handle it client-side? The check could be as simple as checking if the following is defined:
```ts
response?.error
```

**Comment by GufCab** [comment]
> 

This was a frontend issue - fixed in latest push.

File: src/controllers/admin-controller/sigfox/sigfox-group.controller.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (GufCab) [reply src/controllers/admin-controller/sigfox/sigfox-group.controller.ts:84]: Could be implemented that way - but the way it's used it is specifically not an error, it's just used as a check if it exists

**Review by AramAlsabti — APPROVED** [review]

**Pull request merged**